ruby_on_rails/bootstrap.md

```diff
@@ -18,6 +18,7 @@ Add them with:
 and configure them in `environment.js`:
 
 ```js
+const webpack = require('webpack')
```

Comment:
why did you add this? a comment or description would halp me to review the PR.

Reply:
Doesn't the variable `webpack` that is used [here](https://github.com/renuo/applications-setup-guide/blob/22dea33199f9aa77e7f45a04fc437ac0f7aacdc5/ruby_on_rails/bootstrap.md#L22) first have to be required?

Reply:
![image](https://user-images.githubusercontent.com/42026584/56136079-51939080-5f92-11e9-9cc0-6a100fc65e59.png)